pkg/driver/controller_server.go

```diff
@@ -341,6 +341,11 @@ func (driver *Driver) createVolume(
 		return nil, status.Error(codes.Internal, errStr)
 	}
 
+	if val, ok := createParameters[protectionTemplateKey]; ok {
+		log.Infof("protectionTemplate is deprecated for volume create, removing %s=%s from the request", protectionTemplateKey, val)
```

Comment:
This is more than deprecating :)

This is removal.

Reply:
Can we change the verbiage to mention that volumeGroups are used instead and the parameter has "moved"?

Reply:
sure. will update